p28_return_indices.py

```diff
@@ -0,0 +1,33 @@
+Time complexity - O(n)
+space complexity - O(1)
+Ran on leetcode - Yes
+Faced issues - Yes . In python to retrieve the key for that particular value we dont have a inbuilt function whereas this problem can be done easily in java 
+
+Code Description:
+The following problem well in brute force i.eO(n2) but i tried to do it in O(n).I used a dictinary to store all the values in dictionary and while traversing through the list i checked if target-nums[i] is present or not in the list.If yes i return the both the indices.
+
+
+class Solution(object):
+    def dict_search(self,d,i,remain):
+        for key,value in d.items():
+            if key!=i and d[key]==remain:
+                return key
+                break
```

Comment:
return will directly bring the control out. Your break statement is not necessary